docs/source/configuration.rst

```diff
@@ -635,13 +635,17 @@ To ignore users based on their hosts and/or nicks, you can use these options:
 Logging
 =======
 
-Sopel's outputs are redirected to a file named ``<base>.stdio.log``, located in
-the **log directory**, which is configured by :attr:`~CoreSection.logdir`.
+Sopel writes logs of its activities to its **log directory**, which is
+configured by :attr:`~CoreSection.logdir`. Depending on the enabled options,
+there may be as many as four log files per config:
 
-The ``<base>`` prefix refers to the configuration's
-:attr:`~sopel.config.Config.basename` attribute.
+* ``<configname>.sopel.log``: standard logging output
+* ``<configname>.error.log``: errors only
+* ``<configname>.exceptions.log``: exceptions and accompanying tracebacks
+* ``<configname>.raw.log``: raw traffic between Sopel and the IRC server, if
+  enabled (see :ref:`below <Log to a Channel>`)
```

Comment:
I believe it's about Raw Logs, not Log to a Channel?

```suggestion
  enabled (see :ref:`below <Raw Logs>`)
```

Reply:
> Good catch.

Back atcha. Will fix and work on your other review comment.